src/Administration/Resources/app/administration/src/module/sw-first-run-wizard/view/sw-first-run-wizard-welcome/sw-first-run-wizard-welcome.scss

```
@@ -1,27 +1,24 @@
-@import "~scss/variables";
-
 .sw-first-run-wizard-welcome {
-    width: 100%;
-    align-self: flex-start;
-    margin-top: 100px;
-
     .center {
         text-align: center;
     }
 
     .headline-welcome {
-        font-size: 32px;
-        font-weight: normal;
+        margin-top: var(--scale-size-20);
+        margin-bottom: var(--scale-size-8);
+        font-family: var(--font-family-headings);
+        font-size: var(--font-size-2xl);
+        font-weight: var(--font-weight-medium);
+        line-height: var(--font-line-height-3xl);
+        color: var(--color-text-primary-default);
         text-align: center;
-        letter-spacing: 0.2px;
-        line-height: 45px;
     }
 
     .text-welcome {
-        font-size: $font-size-xs;
-        font-weight: normal;
+        font-size: var(--font-size-xs);
+        font-weight: var(--font-weight-regular);
+        margin: 0 auto var(--scale-size-30);
+        width: 550px;
```

Does this really need a fixed width? If so I would also suggest switching to rem or is a specific reason to still use pixels here?
```suggestion
        width: 34.375rem;
```

---

the text is intended to break, thats why it has a fixed width, also the rem number isn' really nice and it's "perfectly sized" so no need to change it

---

> it's "perfectly sized" so no need to change it

But it's perfectly sized for the default font size right? If you increase the font size the text would get larger as well and will have to break earlier. If that is intended this is fine 🤷 .

In general I guess we should test our UI more with different zoom / `font-size` settings and see if everything scales and looks consistent, now that we use more `rem` instead of `px` as the design tokens are using that.